src/Tools/dotnet-monitor/DiagnosticsMonitorCommandHandler.cs

```diff
@@ -2,8 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.AspNetCore.Hosting;
```

Comment:
Shouldn't this entire fire mostly go away except for booting up the WebHost? This logic that you have to bootstrap the diagnostics services should be moved into startup or to CreateWebHostBuilder.

Reply:
Yes, this will definitely need to be cleaned up. I am currently keeping the two service collections separated, with the exception of the one interface that is shared between them. Given that we likely don't need to have a command line scenario, merging them makes sense.